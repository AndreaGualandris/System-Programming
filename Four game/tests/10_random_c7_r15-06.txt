5
2
5
0
1
4
2
4
4
4
6
4
6
6
3
3
5
6
1
1
3
6
5
2
0
5
2
5
4
3
2
5
5
5
2
4
5
4
0
4
5
3
6
4
3
5
3
5
0
5
2
6
5
5
2
0
1
4
5
6
4
6
5
3
0
6
1
1
6
3
2
2
6
1
2
1
6
4
5
3
2
0
4
1
5
6
4
3
2
3
2
2
2
5
4
6
4
5
1
1
5
5
5
6
4
