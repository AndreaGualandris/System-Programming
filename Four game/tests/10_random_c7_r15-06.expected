-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
       
-------
0123456
Player 'o' to move:
     o
     v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
     o 
-------
0123456
Player 'x' to move:
  x
  v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
  x  o 
-------
0123456
Player 'o' to move:
     o
     v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
     o 
  x  o 
-------
0123456
Player 'x' to move:
x
v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
     o 
x x  o 
-------
0123456
Player 'o' to move:
 o
 v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
     o 
xox  o 
-------
0123456
Player 'x' to move:
    x
    v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
     o 
xox xo 
-------
0123456
Player 'o' to move:
  o
  v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
  o  o 
xox xo 
-------
0123456
Player 'x' to move:
    x
    v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
  o xo 
xox xo 
-------
0123456
Player 'o' to move:
    o
    v
-------
       
       
       
       
       
       
       
       
       
       
       
       
    o  
  o xo 
xox xo 
-------
0123456
Player 'x' to move:
    x
    v
-------
       
       
       
       
       
       
       
       
       
       
       
    x  
    o  
  o xo 
xox xo 
-------
0123456
Player 'o' to move:
      o
      v
-------
       
       
       
       
       
       
       
       
       
       
       
    x  
    o  
  o xo 
xox xoo
-------
0123456
Player 'x' to move:
    x
    v
-------
       
       
       
       
       
       
       
       
       
       
    x  
    x  
    o  
  o xo 
xox xoo
-------
0123456
Player 'o' to move:
      o
      v
-------
       
       
       
       
       
       
       
       
       
       
    x  
    x  
    o  
  o xoo
xox xoo
-------
0123456
Player 'x' to move:
      x
      v
-------
       
       
       
       
       
       
       
       
       
       
    x  
    x  
    o x
  o xoo
xox xoo
-------
0123456
Player 'o' to move:
   o
   v
-------
       
       
       
       
       
       
       
       
       
       
    x  
    x  
    o x
  o xoo
xoxoxoo
-------
0123456
Player 'x' to move:
   x
   v
-------
       
       
       
       
       
       
       
       
       
       
    x  
    x  
    o x
  oxxoo
xoxoxoo
-------
0123456
Player 'o' to move:
     o
     v
-------
       
       
       
       
       
       
       
       
       
       
    x  
    x  
    oox
  oxxoo
xoxoxoo
-------
0123456
Player 'x' to move:
      x
      v
-------
       
       
       
       
       
       
       
       
       
       
    x  
    x x
    oox
  oxxoo
xoxoxoo
-------
0123456
Player 'o' to move:
 o
 v
-------
       
       
       
       
       
       
       
       
       
       
    x  
    x x
    oox
 ooxxoo
xoxoxoo
-------
0123456
Player 'x' to move:
 x
 v
-------
       
       
       
       
       
       
       
       
       
       
    x  
    x x
 x  oox
 ooxxoo
xoxoxoo
-------
0123456
Player 'o' to move:
   o
   v
-------
       
       
       
       
       
       
       
       
       
       
    x  
    x x
 x ooox
 ooxxoo
xoxoxoo
-------
0123456
Player 'x' to move:
      x
      v
-------
       
       
       
       
       
       
       
       
       
       
    x x
    x x
 x ooox
 ooxxoo
xoxoxoo
-------
0123456
Player 'o' to move:
     o
     v
-------
       
       
       
       
       
       
       
       
       
       
    x x
    xox
 x ooox
 ooxxoo
xoxoxoo
-------
0123456
Player 'o' wins!
